consoleme/celery_tasks/celery_tasks.py

```diff
@@ -573,8 +573,26 @@ def cache_policies_table_details() -> bool:
     if all_s3_errors:
         s3_errors = json.loads(all_s3_errors)
 
+    allowed_tags = config.get("roles.allowed_tags", {})
+    allowed_arns = config.get("roles.allowed_arns", [])
     for arn, role_details_j in all_iam_roles.items():
         role_details = ujson.loads(role_details_j)
+        role_details_policy = ujson.loads(role_details.get("policy"))
```

Comment:
I think this should have a default value to prevent a json loading error, ie:

`ujson.loads(role_details.get("policy", {))`

Reply:
added